src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+abstract class Figure implements AreaCalculator, DrawInfo {
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the empty line before the class declaration.